tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -340,7 +317,7 @@ def create_snapshot(self, snap_num):
         """
         log.info(f"Taking snapshot number {snap_num}")
         # Getting UTC time before test starting for log retrieve
-        UTC_datetime = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+        start_time = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
can be replace with 
`start_time = self.get_time("csi")`

Reply:
Done, thank you.